test/resources/layout.css

```diff
@@ -62,6 +62,7 @@ body {
 
 .btn-playground,
 .btn-playground--compact,
+.btn-playground--transparent,
 .btn-playground--emphasized {
   border: 1px solid black;
   border-radius: 4px;
```

Comment:
`rem` value should be used instead of `px`. There are other places with the same issue